src/classes/ExtendedGraph.class.php

```diff
@@ -1643,7 +1783,7 @@ function get_label_for_uri($uri)
         }
         if (!isset(self::$labelProperties))
         {
-            throw new TripodException('Please initialise ExtendedGraph::$labelProperties');
+            throw new \Tripod\Exceptions\Exception('Please initialise ExtendedGraph::$labelProperties');
```

Comment:
This is the generic exception for Tripod.  I guess you could call it a base exception, but it's not like, say, an abstract base class.

It's intended to differentiate exceptions thrown within Tripod (that don't necessarily require their own fit-for-purpose Exception class) from general exceptions elsewhere.